README.md

```diff
@@ -54,43 +62,26 @@ See the output of `deepspeech -h` for more information on the use of `deepspeech
 - [Contribution guidelines](#contribution-guidelines)
 - [Contact/Getting Help](#contactgetting-help)
 
-## Prerequisites
-
-* [Python 3.6](https://www.python.org/)
-* [Git Large File Storage](https://git-lfs.github.com/)
-* Mac or Linux environment
-* Go to [build README](examples/net_framework/README.md) to start building DeepSpeech for Windows from source.
-
-## Getting the code
-
-Install [Git Large File Storage](https://git-lfs.github.com/) either manually or through a package-manager if available on your system. Then clone the DeepSpeech repository normally:
-
-```bash
-git clone https://github.com/mozilla/DeepSpeech
-```
-
-
 ## Using a Pre-trained Model
 
-There are three ways to use DeepSpeech inference:
+Inference using a DeepSpeech pre-trained model can be done with a client/language binding package. We have four clients/language bindings in this repository, listed below, and also a few community-maintained clients/language bindings in other repositories, listed [further down in this README](#third-party-bindings).
 
-- [The Python package](#using-the-python-package)
-- [The Node.JS package](#using-the-nodejs-package)
+- [The Python package/language binding](#using-the-python-package)
+- [The Node.JS package/language binding](#using-the-nodejs-package)
 - [The Command-Line client](#using-the-command-line-client)
+- [The .NET client/language binding](native_client/dotnet/README.md)
 
-Running `deepspeech` might require some runtime dependencies to be already installed on your system. Regardless of which bindings you are using, you will need the following:
+Running `deepspeech` might require some runtime dependencies to be already installed on your system:
 
-* libsox2
-* libstdc++6
-* libgomp1
-* libpthread
+* sox - The Python and Node.JS clients use SoX to resample files to 16kHz.
+* libpthread - On Linux, some people have had to install libpthread manually.
```

Comment:
I think we want to keep libstdc++ and libgomp ?

Reply:
And maybe we should add GLIBC versions ?

Reply:
Oh, I didn't mean to remove that, will fix.